beetsplug/bpd/gstplayer.py

```diff
@@ -57,8 +62,19 @@ def __init__(self, finished_callback=None):
 
         # Set up the Gstreamer player. From the pygst tutorial:
         # http://pygstdocs.berlios.de/pygst-tutorial/playbin.html
-        self.player = gst.element_factory_make("playbin2", "player")
-        fakesink = gst.element_factory_make("fakesink", "fakesink")
+        ####
+        # Updated to GStreamer 1.0 with:
+        # https://wiki.ubuntu.com/Novacut/GStreamer1.0
+        self.player = Gst.ElementFactory.make("playbin", "player")
+
+        if self.player is None:
+            raise RuntimeError("Could not create playbin")
```

Comment:
For these user-visible errors, please use `beets.ui.UserError` instead of the built-in Python `RuntimeError` for readable error messages.